sycl/test/extensions/forward_progress_L0_gpu.cpp

```diff
@@ -0,0 +1,45 @@
+// REQUIRES: level_zero, gpu
```

Comment:
Any tests which require actual HW should be in `e2e` folder. You should also use `%{build}` and `%{run}` substitutions in there